pyiron/atomistics/master/elastic.py

```diff
@@ -0,0 +1,385 @@
+# coding: utf-8
+# Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
+# Distributed under the terms of "New BSD License", see the LICENSE file.
+
+from __future__ import print_function
+
+import numpy as np
+import scipy.constants
+from pyiron.atomistics.master.parallel import AtomisticParallelMaster
+from pyiron_base import JobGenerator
+from sklearn.linear_model import LinearRegression
+from collections import defaultdict
+import warnings
+import itertools
+
+__author__ = "Joerg Neugebauer, Jan Janssen"
+__copyright__ = (
+    "Copyright 2020, Max-Planck-Institut für Eisenforschung GmbH - "
+    "Computational Materials Design (CM) Department"
+)
+__version__ = "1.0"
+__maintainer__ = "Jan Janssen"
```

Comment:
Hmm, I might need to steal this idea myself... write some code and then slap Jan down as the maintainer 😝